app/models/account.rb

```diff
@@ -45,6 +45,7 @@
 #  moved_to_account_id     :integer
 #  featured_collection_url :string
 #  fields                  :jsonb
+#  media_uploads_disabled  :boolean          default(FALSE), not null
```

Comment:
Is there a good reason to put this on `accounts` which includes remote people, rather than `users` which stores `admin` and `moderator` booleans?

Also, I am concerned that a single boolean for this one type of permission is too narrow and will have to be changed with a lot of effort somewhere down the line. This is entering a flexible-role territory, the kinda things that usually get stored as either bitmasks, or separate records, or... well, sometimes separate booleans, but I think more care and discussion needs to be had before approving this